specification/types.md

```diff
@@ -94,12 +94,19 @@ A structure which supports definition of arbitrary properties, with keys of type
 
 This structure is populated by a provider for use by an [Application Author](./glossary.md#application-author) (via the [Evaluation API](./glossary.md#evaluation-api)) or an [Application Integrator](./glossary.md#application-integrator) (via [hooks](./sections/04-hooks.md)).
 
+### Event Metadata
+
+A structure supporting the addition of arbitrary event data.
+It supports definition of arbitrary properties, with keys of type `string`, and values of type `boolean`, `string`, or `number`.
+It defines an `error message` field with a value of type `string`.
```

Comment:
Should the `error message` only be a defined field in `PROVIDER_ERROR` events metadata or in any?
Could also be in any, so that if one likes it could also be included in `PROVIDER_STALE` events, but I would rather only _define_ the field for `PROVIDER_ERROR`.

Reply:
But not sure if this is important for the spec, just wondering.

Reply:
> Should the error message only be a defined field in PROVIDER_ERROR events metadata or in any event types metadata?

I've added `error event metadata`, see [delta](https://github.com/open-feature/spec/pull/182/commits/0c0dbc111c65dbe418250b22c5b67f651b0124a5).

Reply:
Nice thats good!